### DISCUSSION TITLE
How do you change rodauth views labels?

### DISCUSSION THREAD

**Discussion by jgthms** [janko/rodauth-rails#265] [Q&A] [answered]
After generating the views with `rodauth:views`, I can see that a lot of labels come from the config: `rodauth.login_label`, `rodauth.password_label` etc.

How do you change these?

(I only want to change a few of them, not translate them).

**Comment by janko** [comment, marked as answer]
See [this section](https://github.com/jeremyevans/rodauth?tab=readme-ov-file#label-Basic+Usage) for "auth methods" and "auth value methods". These are auth *value* methods, so you can pass a string directly in the configuration DSL:

```rb
class RodauthMain < Rodauth::Rails::Auth
  configure do
    login_label "..."
    password_label "..."
  end
end
```

But you don't need to use these methods, you can just hardcode strings directly in view templates, like you'd normally do in Rails. The main reason rodauth-rails generates templates that use these methods is so that things like i18n works automatically.

**Reply by jgthms** [reply]
Oh right, it's a _method_. I initially tried `login_label = "My label"` but it failed indeed.

Many thanks for the quick response.
